### PR TITLE
Development

### PR DESCRIPTION
**Pull request by KaushikShivam**
We have done the validation. Looking forward to your feedback.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+linters:
+  rubocop:
```
Comment (mosaaleb) [inline .stickler.yml]: Rubocop is linter for ruby code, all we need in this file is to enable the [style lint](https://github.com/microverseinc/linters-config/tree/master/css)

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Just Perfect @KaushikShivam, @rociac :100:

#### Well done team. Your code is perfect and looks professional. The UI is exact replica :clap:. I liked your class naming convention, this is BEM naming convention, right?. The code is organized and easy to read.

One last thing, we just need to add style lint to stickler file and make sure that all checks have passed, see my comment. Also let's fill the readme file properly with [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).

Keep up the good work team.

Once you finish these minor requirements, please submit a new code review.

Happy coding :computer:

**Comment by KaushikShivam** [comment]
Thanks. We've fixed the changes. Also, thanks a lot for the kind review and feedback.

**Review by mosaaleb — APPROVED** [review]
## Well done team :clap: 

You can now merge to master :+1: . Also don't forget to submit a project completion. 

Good luck with the projects to come. Happy coding :computer:

**Pull request merged**